ocp_resources/resource.py

```diff
@@ -364,6 +364,7 @@ class ApiGroup:
         UPLOAD_CDI_KUBEVIRT_IO = "upload.cdi.kubevirt.io"
         V2V_KUBEVIRT_IO = "v2v.kubevirt.io"
         VM_KUBEVIRT_IO = "vm.kubevirt.io"
+        VELERO_IO = "velero.io"
```

Comment:
Please move it after `V2V_KUBEVIRT_IO` (alphabetic order)

Reply:
Done